Packs/SecneurXAnalysis/Integrations/SecneurXAnalysis/SecneurXAnalysis.yml

```
@@ -0,0 +1,251 @@
+    execution: false
+    name: snx-analysis-submit-file
+    outputs:
+    - contextPath: SnXAnalysis.SubmitFile.task_uuid
+      description: Task UUID is unique ID for submitted file. Use this ID for get the report and verdict.
+      type: String
+    - contextPath: SnXAnalysis.SubmitFile.submission_time
+      description: Submission Time Created at the time the file was submitted
+      type: String
+  - arguments:
+    - default: false
+      description: Input the URL for analysis
+      isArray: false
+      name: URL
+      required: true
+      secret: false
+    - default: false
+      description: Type the priority of the sample for analysis. Default is set to Normal. Possible values are High, Normal
+      isArray: false
+      name: Priority
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - "High"
+      - "Normal"
+    - default: false
+      description: Type the duration of the analysis in seconds. Not all malicious programs are active right after the launch.
+        Some of them take time to fully reveal the attack vectors. Default is set to 120. Possible values are 120, 180, 240, 300
+      isArray: false
+      name: Duration
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - "120"
+      - "180"
+      - "240"
+      - "300"
+    - default: false
+      description: Reboot the system during the analysis. Default is set to 'False'. Possible values are True, False
+      isArray: false
+      name: Reboot
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - "True"
+      - "False"
+    deprecated: false
+    description: Submit the URL for Analysis
+    execution: false
+    name: snx-analysis-submit-url
+    outputs:
+    - contextPath: SnXAnalysis.SubmitURL.task_uuid
+      description: Task UUID is the unique ID for the submitted file. Use this ID to get the report.
+      type: String
+    - contextPath: SnXAnalysis.SubmitURL.submission_time
+      description: Submission Time Created at the time the file was submitted
+      type: String
+  - arguments:
+    - default: false
+      description: Input the Task UUID value obtained as response from submission
+      isArray: false
+      name: task_uuid
+      required: true
+      secret: false
+    - default: false
+      description: Mention the output format of the report. The value should be "json" or "html". The default is set to "json"
+      isArray: false
+      defaultValue: "json"
+      name: report_format
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - "html"
+      - "json"
+    deprecated: false
+    description: Get the detailed report of the analyzed sample.
+    execution: false
+    name: snx-analysis-get-report
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.5.31928
+fromversion: 5.5.0
+tests:
+- No tests (auto formatted)
```

After you change the IDs of the TestPlaybooks, Please put the ids here.

```suggestion
- Detonate File - SecneurX Analysis - Test
- Detonate URL - SecneurX Anlaysis - Test
```